images/linux/scripts/installers/swift.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+################################################################################
+##  File:  swift.sh
+##  Desc:  Installs Swift
+################################################################################
+
+# Install # Install
+image_label="$(lsb_release -rs)"
+wget https://swift.org/builds/swift-5.1.5-release/ubuntu${image_label//./}/swift-5.1.5-RELEASE/swift-5.1.5-RELEASE-ubuntu$image_label.tar.gz
+tar xzf swift-5.1.5-RELEASE-ubuntu$image_label.tar.gz
+sudo mv swift-5.1.5-RELEASE-ubuntu$image_label /usr/share/swift
+sudo echo "PATH=/usr/share/swift/usr/bin:$PATH" | sudo tee -a /etc/environment
+source  ~/.bashrc
```

Comment:
Probably, we should not add this folder to the path because it will override a lot of default tools like [clang and others](https://buildcanary.visualstudio.com/CanaryBuilds/_build/results?buildId=836255&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=4d2ea636-6e6e-5831-8b5a-8ce5e9ce8bbc).
Probably, we should make symlink `/usr/share/swift/usr/bin/swift -> /usr/local/bin/swift` and also inroduce some environment variable like `SWIFT_PATH` with `/usr/share/swift/usr/bin`